### ISSUE TITLE
kots pull helmchart should not require a license

### ISSUE THREAD

**Issue by elisiano** [replicatedhq/kots#383]
As discussed over [slack](https://kubernetes.slack.com/archives/CFFKXV0JZ/p1583505887006800) this is not the expected behavior.

```
epetrini@epetrini-mac:~/temp/kots$ kubectl kots pull helm://stable/elasticsearch

Error: failed to parse license from file: failed to read license file: read /Users/epetrini/temp/kots: is a directory
epetrini@epetrini-mac:~/temp/kots$ kubectl kots version
Replicated Kots 1.13.4
```

**Comment by dexhorthy** [comment]
Thanks for filing this @elisiano ! CC @marccampbell

**Comment by marccampbell** [comment]
@laverya this was fixed in 1.13.4 (one it was released)?

**Comment by marccampbell** [comment]
Closing as fixed in #386. Thanks for the report!
